NetKAN/SlingShotter.netkan

```diff
@@ -0,0 +1,22 @@
+{
+    "spec_version": "v1.16",
+    "identifier": "SlingShotter",
+    "$kref": "#/ckan/github/johnfink8/KerbalSlingshotter",
+    "name"        : "Kerbal SlingShotter",
+    "author"      : [ "Overengineer1" ],
+    "abstract"    : "Display positions in the map for ships, planets, and moons at a selected time.",
+    "version"     : "1.0.0",

```

Comment:
`version` should not be hardcoded as 1) all releases of the mod on github will be tagged with the same version 2) netkan can scrape the mod's version information from the github release.